src/14/HW/Note.ts

```diff
@@ -0,0 +1,36 @@
+export class Note {
```

Comment:
There were two types of notes in requirements. I can't find them. Please, double-check it.